v3/integrations/nrgin/go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/newrelic/go-agent/v3/integrations/nrgin
 
-// As of Dec 2019, the gin go.mod file uses 1.12:
+// As of Jul 2022, the gin go.mod file uses 1.18:
 // https://github.com/gin-gonic/gin/blob/master/go.mod
-go 1.12
+go 1.18
```

Comment:
Does this need to be 1.18?

Reply:
I'm not sure if it's _required_, but I switched it in order to match the go version in the [gin/go.mod](https://github.com/gin-gonic/gin/blob/master/go.mod) file.

Reply:
Is that version of gin the version that is required for it to work again? If so then this is fine.

Reply:
Did some testing and it looks like we actually don't. I will lower the version back to 1.12 now.